doc/sphinx/espressopp.io.DumpH5MDParallel.rst

```diff
@@ -0,0 +1,38 @@
+Parallel H5MD Output
+====================
+
+H5MD_ is a format specification on top of the HDF5_ file format.
+
+.. _H5MD: https://nongnu.org/h5md/
+.. _HDF5: https://www.hdfgroup.org/solutions/hdf5/
+
+Dump Simulation
+---------------
+
+Dumping a simulation with ``espressopp.io.DumpH5MDParallel``
```

Comment:
Am I right that this is only to dump a single frame during the simulation?

Reply:
Ok, so to be clear, the following code:

```python
for step in range(100):
    integrator.run(100)
    h5mdump.dump()
```
will result with 100 frames or only a single last frame? If there will be only a single frame then I would put this information explicitly in documentation. Other dumps like xyz, pdb or gro behave differently, storing multiple frames.